Dockerfile

```diff
@@ -1,38 +1,28 @@
-FROM ubuntu:14.04.4
+FROM ubuntu:14.04
```

Comment:
According to https://hub.docker.com/_/ubuntu/, :14.04 will always point to the latest release of ubuntu 14.04 (currently it points to 14.04.5).